go.sum

```diff
@@ -83,6 +83,9 @@ github.com/grailbio/base v0.0.5 h1:9LoafGwmMR4QPEOZ5Kz0posu/pbGhDleT7Owqj8l7Yg=
 github.com/grailbio/base v0.0.5/go.mod h1:OFVz7zmqb1D+Jbew0B4DCIpl4ozzVFxf+JKQZBBIQzE=
 github.com/grailbio/bigmachine v0.5.5 h1:uHdPrVTKw9BQmcfE70b7q126LJac8TiCnqOvS+UmvOo=
 github.com/grailbio/bigmachine v0.5.5/go.mod h1:8cYMHQBaSMyR9Gy9vh6YDE6uo+SgbEsnBXIGVp3gKGE=
+github.com/grailbio/testutil v0.0.1/go.mod h1:j7teGaXqRY1n6m7oM8oy954lxL37Myt7nEJZlif3nMA=
+github.com/grailbio/testutil v0.0.3 h1:Um0OOTtYVvyxwQbO48K3t6lNmLPY4sL3Vn6Sw0srNy8=
+github.com/grailbio/testutil v0.0.3/go.mod h1:f9+y7xMXeXwyNcdV5cmo6GzRiitSOubMmqcqEON7NQQ=
```

Comment:
This is causing the tests to fail.

Reply:
this testutil thing is annoying:-( Hopefully it gets fixed shortly in the go toolchain.